content/en/data_streams/technologies/azure_service_bus.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Azure Service Bus for Data Streams Monitoring
+---
+
+### Prerequisites
+
+* [Datadog Agent v7.34.0 or later][1]
+
+{{% data_streams/monitoring-azure-service-bus %}}
+
+### Support for Azure Service Bus in Data Streams Monitoring
+
+<table>
```

Comment:
decided to use html table styling for these for consistency. Some of the tech tables have cells that span multiple columns (kafka's Go language has multiple libraries for ex.)